.gitignore

```diff
@@ -29,4 +31,4 @@ flamegraph.svg
 perf.data
 perf.data.old
 app/release
-release.env
\ No newline at end of file
+app/release.env
```

Comment:
this I think was right before, I kept `release.env` in the root directory with the ACRA credentials

Reply:
would it be ok to add *.idea to avoid commiting these? https://i.imgur.com/4NNH8id.png

Reply:
It looks like [jetbrains says](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) to ignore those, but not to ignore everything